### DISCUSSION TITLE
Database size

### DISCUSSION THREAD

**Discussion by Fawesum** [plausible/analytics#3492] [Self-Hosted Support]
Just wondering if this is normal. After approx. one day of using Plausible my database is already at 120 MB. Around 13k pageviews. This sounds pretty extreme to me.

Am I really going to be looking at 44 GB of yearly logs here for one site?

**Comment by ruslandoga** [comment]
👋 @Fawesum 

These might be helpful:

- https://github.com/plausible/hosting/issues/84
- https://github.com/plausible/analytics/discussions/773
- https://github.com/plausible/analytics/discussions/2566

If not, please share your configuration and describe your deployment in more detail.
